examples/k8s/k7e/community_day_2019_may/client-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: client
-          image: gcr.io/spiffe-io/spire-agent:1.5.0
+          image: ghcr.io/spiffe/spire-agent:1.5.1
           imagePullPolicy: Always
           command: ["/bin/sleep", "1d"]
```

Comment:
this sleep will start failing since we are moving to scratch image, can you replace it with agent watch?